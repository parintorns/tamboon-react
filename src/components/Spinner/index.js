import React from 'react'

import { Wrapper } from './styles'

export default function Spinner() {
  return <Wrapper>Loading...</Wrapper>
}
