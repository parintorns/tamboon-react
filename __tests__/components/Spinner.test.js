import React from 'react'
import { shallow } from 'enzyme'

import Spinner from '../../src/components/Spinner'

let wrapper

const props = {}

beforeEach(() => {
  wrapper = shallow(<Spinner {...props} />)
})

describe('Render', () => {
  it('should render Spinner', () => {
    expect(wrapper.debug()).toMatchSnapshot()
  })
})
