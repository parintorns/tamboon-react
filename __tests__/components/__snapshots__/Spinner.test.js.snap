// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render should render Spinner 1`] = `
"<div>
  Loading...
</div>"
`;
